### ISSUE TITLE
unmet load hours

### ISSUE THREAD

**Issue by weilixu** [pnnl/ruleset-checking-tool#995]
Rule [19-5](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section19/Rule19-5.md) and [19-6](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section19/Rule19-6.md) checks unmet load hours.

The[ output schema](https://github.com/open229/ruleset-model-description-schema/blob/initial-RCT-output-report-schema/docs229/Output2019ASHRAE901.schema.json#L150-L158) sets the unmet load hours as required data.

In that case, do you agree that unmet_load_hours_heating, unmet_load_hours_cooling and coincident_unmet_load_hours cannot be Null since these three parameters are required data and their types are number?

**Comment by mkarpman** [comment]
It seems that the intent is to allow populating either unmet_hours OR (unmet_heating AND unmet_cooling). 
- If unmet_hours are specified, than we can readily check whether the project meets the min. 300 hrs requirement. 
- If this field is blank, then we can be sure that the project passes ONLY of unmet_heating + unment_cooling <=300 (i.e., your best case scenario occurs). In all other cases, project may fail the limit.

**Comment by JacksonJ-KC** [comment]
That sounds fine to me. The logic needs to be adjusted to prioritize unmet_hours and only fall back to unmet_heating + unmet_cooling when unmet_hours is not defined. Agreed?

**Comment by mkarpman** [comment]
I agree, but I wonder if othres have different recollection on how these data elements are related. In either case, notes to the schema should be updated to make this clear. E.g., I think it would be sufficient to provide either one or the other, and not both.

**Comment by weilixu** [comment]
The current rule is updated with the suggested logic. Closing this issue.
